### DISCUSSION TITLE
The "Application" page does not display all available applications.

### DISCUSSION THREAD

**Discussion by IxeYgrek** [cockpit-project/cockpit#20590] [Q&A]
Hello,

When I go to the "Applications" page, I only see the two applications currently installed, without being able to install others. I've seen this problem mentioned a lot, but all the issues I've seen on Github about it are closed because the problem has been fixed. Except that I still have this problem.

I've obviously clicked on the refresh button at the top of the page but it doesn't change anything

I'm using Ubuntu 22.04.4 LTS and I installed Cockpit following the documentation and using the backport option.
The version of Cockpit installed is 316-1~bpo22.04.1 but I see that on the site the latest version of Cockpit is 318.
So does this mean that with Ubuntu 22.04 I can't upgrade to Cockpit version 318 and that this bug hasn't been fixed on version 316 supported for Ubuntu 22.04?

Thanks in advance for your help.

Regards,
IxeYgrek

**Comment by mmosquera91** [comment]
I have the exact same issue you're describing and I'm also on Ubuntu 22.04 LTS and cockpit 333

**Reply by Venefilyn** [reply]
Could you open a bug issue? Latest version having this bug is definitely something we want to track and fix

**Reply by jelly** [reply]
This is likely https://github.com/cockpit-project/cockpit/issues/20354

**Reply by begunfx** [reply]
I have this issue as well.  Running on latest Raspbian OS (Debian).  In case it's relevant, I installed cockpit using the Debian Backports Repository.  I used the following tutorial to get it up and running:
[pimylifeup.com](https://pimylifeup.com/raspberry-pi-cockpit/)

I'm running the latest version of cockpit: 335-2~bpo12+1

**Reply by Venefilyn** [reply]
@begunfx check out #20354 and see if you can find any logs as well. The more details we have the better :)

**Reply by begunfx** [reply]
> @begunfx check out #20354 and see if you can find any logs as well. The more details we have the better :)

Thanks @Venefilyn.  I posted in the other issue you shared.  I checked the logs but unfortunately didn't see any errors or anything that seemed relevant to the plugins.

**Comment by bernhardberger** [comment]
I also can't see any applications except the preinstalled "Networking" and "Storage".

Installed cockpit from the Ubuntu 26.04 Repo (daily build).
